src/main/resources/roleconfig/role_common.json

```diff
@@ -1691,7 +1692,8 @@
         },
         "attributes": {
           "jurisdiction": {
-            "mandatory": true
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
```

Comment:
no need to add jurisdiction to special access cases. please remove "values": ["PUBLICLAW"]